.travis.yml

```diff
@@ -10,18 +10,21 @@ rvm:
 - &oldest_ruby 2.3.8
 - jruby-9.2.7.0
 - jruby-9.1.17.0
-# the test suite currently crashes on truffleruby
-#- truffleruby-rc13
 matrix:
   include:
   - rvm: *oldest_ruby
     env: ROUGE_VERSION='~> 2.0.0'
+  - name: "TruffleRuby"
```

Comment:
Is there a reason you're defining the name property here?

Reply:
No, I've removed it.